lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1684,19 +1684,20 @@ def jsonrpc_file_set_status(self, status, **kwargs):
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
     @AuthJSONRPCServer.flags(delete_target_file='-f', delete_all='-a')
-    def jsonrpc_file_delete(self, delete_target_file=True, delete_all=False, **kwargs):
+    def jsonrpc_file_delete(self, delete_target_file=False, delete_all=False, **kwargs):
         """
         Delete a LBRY file
 
         Usage:
-            file_delete [-a | -f] [--sd_hash=<sd_hash>] [--file_name=<file_name>]
+            file_delete [-a] [-f] [--sd_hash=<sd_hash>] [--file_name=<file_name>]
                         [--stream_hash=<stream_hash>] [--claim_id=<claim_id>]
                         [--outpoint=<outpoint>] [--rowid=<rowid>]
                         [--name=<name>]
 
         Options:
-            -a                          : delete file from downloads and delete stored blobs
-            -f                          : delete only from downloads, do not delete blobs
+            -a, --delete_all            : if there are multiple matching files, allow the deletion
```

Comment:
can we get rid of the short `-a` option, and leave only the long option. i dont want anyone to accidentally delete all their files